src/test/java/unicash/model/transaction/DateTimeTest.java

```diff
@@ -124,4 +125,16 @@ public void hashCode_test() {
         assertEquals(dateTime1.hashCode(), dateTime2.hashCode());
         assertNotEquals(dateTime1.hashCode(), dateTime3.hashCode());
     }
+
+    @Test
+    public void getYearMonth_test() {
```

Comment:
This unit test name could be more descriptive like `getYearMonth_equalsOtherYearMonth`